src/containers/Layout/useGetMenuItems.tsx

```diff
@@ -118,7 +118,7 @@ const useGetMenuItems = () => {
     }
 
     return menuItems;
-  }, [accountAddress]);
+  }, [accountAddress, chainId]);
```

Comment:
I think the actual dependencies are `corePoolRouteEnabled`, `historyRouteEnabled`, `convertVrtRouteEnabled` and `vaiRouteEnabled`. Technically they do rely on `chainId`, but there could be instances where even if the chain changes these don't (for example if we were to switch from BSC mainnet to BSC testnet).